prometheus-exporters/install-exporter-amd64.sh

```diff
@@ -385,6 +401,18 @@ DATA_SOURCE_NAME="postgresql://postgres:postgres123@localhost:5432/?sslmode=disa
 EOF
   fi
 
+  if [ "$EXPORTER" == "rds" ]; then
+    cat << EOF > /etc/opsverse/exporters/rds/config.yaml
+instances:
+  - region: <region>
+    instance: <instance>
+    aws_access_key: <aws_access_key>
+    aws_secret_key: <aws_secret_key>
```

Comment:
The user will need to edit this file and then restart the `rds-exporter` service. 

Can we please add a `doc` page for configuring this exporter?

Reply:
Yeah Arul
As this was the high priority for Snapmint, I thought I would add docs later.

Reply:
I will work on doc page.